Frameworks/CoreFoundation/Locale.subproj/CFCalendar.c

```diff
@@ -746,6 +746,12 @@ static CFRange __CFCalendarGetRangeOfUnit2(CFCalendarRef calendar, CFCalendarUni
     // WINOBJC: Add cases for weekOf units. ICU will handle them below.
     case kCFCalendarUnitWeekOfYear:
     case kCFCalendarUnitWeekOfMonth:
+    // WINOBJC: Asking for the range of WeekOfMonth in Year always returns 6.
```

Comment:
OK, but that sounds like a bug in the calculate logic, or maybe in ICU underneath it. Do you know why you were getting that result? I would prefer to fix the error at the source if possible.

---
In reply to: [74342094](https://github.com/Microsoft/WinObjC/pull/844#discussion_r74342094) [](ancestors = 74342094,74336418)